README.md

```diff
@@ -221,16 +221,15 @@ Much of the skill in writing unmaintainable code is the art of camouflage, hidin
 Include sections of code that are commented out but at first glance do not appear to be.
 
 ```js
-for(j=0; j<array_len; j+ =8)
-{
-total += array[j+0 ];
-total += array[j+1 ];
-total += array[j+2 ]; /* Main body of
-total += array[j+3 ]; * loop is unrolled
-total += array[j+4 ]; * for greater speed.
-total += array[j+5 ]; */
-total += array[j+6 ];
-total += array[j+7 ];
+for(j=0; j<array_len; j+ =8) {
+    total += array[j+0 ];
+    total += array[j+1 ];
+    total += array[j+2 ]; /* Main body of
+    total += array[j+3 ]; * loop is unrolled
+    total += array[j+4 ]; * for greater speed.
+    total += array[j+5 ]; */
+    total += array[j+6 ];
+    total += array[j+7 ];
```

Comment:
Can you make all these brackets close on the edge of `[j+{index}]`, not `[j+{index} ]`?